### ISSUE TITLE
Accessibility: Screen Readers: Some content that is visually hidden isn't hidden for screen readers

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#2000]
This issue has been split out from: https://github.com/Automattic/wp-calypso/issues/697

Some content that is visually hidden isn't hidden for screen readers. That means they can access items that they shouldn't be able to at certain times. These include the accordions in the sidebar and the web preview. Those items should be `display: none;` until they are brought onscreen by user action. Otherwise, it's very confusing.

**Comment by rachelmcr** [comment]
Confirmed this is still an issue.

#bug-scrub #pre-gutenburg-bug-purge
